src/test/java/org/tarantool/jdbc/JdbcTypesIT.java

```diff
@@ -74,113 +77,225 @@ static void teardownEnv() throws SQLException {
+        testHelper.executeSql(getCreateTypeTableSQL(targetTypes));
+
+        new TarantoolTestTypeHelper<>(connection, "test_types", BigInteger.class)
+            .setColumns(targetTypes)
+            .setValues(BIGINT_VALS)
+            .testTypes();
+    }
+
+    @Test
+    void testSetBigInteger() throws SQLException {
+        assumeMinimalServerVersion(testHelper.getInstanceVersion(), ServerVersion.V_2_2);
+        TarantoolSqlType[] targetTypes = { TarantoolSqlType.INT, TarantoolSqlType.INTEGER };
+        testHelper.executeSql(getCreateTypeTableSQL(targetTypes));
+
+        new TarantoolTestTypeHelper<>(connection, "test_types", BigInteger.class)
+            .setColumns(targetTypes)
+            .setValues(BIGINT_VALS)
+            .testTypes();
+    }
+
     @Test
     void testSetString() throws SQLException {
+        TarantoolSqlType[] targetTypes = { TarantoolSqlType.VARCHAR, TarantoolSqlType.TEXT };
+        testHelper.executeSql(getCreateTypeTableSQL(targetTypes));
+
         new TarantoolTestTypeHelper<>(connection, "test_types", String.class)
-            .setColumns(TarantoolSqlType.VARCHAR, TarantoolSqlType.TEXT)
+            .setColumns(targetTypes)
             .setValues(STRING_VALS)
             .testTypes();
     }
 
+    @Test
+    void testSetBoolean() throws SQLException {
+        assumeMinimalServerVersion(testHelper.getInstanceVersion(), ServerVersion.V_2_2);
+
+        TarantoolSqlType[] targetTypes = { TarantoolSqlType.BOOLEAN, TarantoolSqlType.BOOL };
+        testHelper.executeSql(getCreateTypeTableSQL(targetTypes));
+
+        new TarantoolTestTypeHelper<>(connection, "test_types", Boolean.class)
+            .setColumns(targetTypes)
+            .setValues(BOOLEAN_VALS)
+            .testTypes();
+    }
+
     @Test
     void testSetFloat() throws SQLException {
+        assumeServerVersionLessThan(testHelper.getInstanceVersion(), ServerVersion.V_2_2);
+
+        TarantoolSqlType[] targetTypes = { TarantoolSqlType.REAL };
+        testHelper.executeSql(getCreateTypeTableSQL(targetTypes));
+
         new TarantoolTestTypeHelper<>(connection, "test_types", Float.class)
-            .setColumns(TarantoolSqlType.REAL)
+            .setColumns(targetTypes)
             .setValues(FLOAT_VALS)
             .testTypes();
     }
 
     @Test
     void testSetDouble() throws SQLException {
+        assumeServerVersionLessThan(testHelper.getInstanceVersion(), ServerVersion.V_2_2);
+
+        TarantoolSqlType[] targetTypes = { TarantoolSqlType.FLOAT, TarantoolSqlType.DOUBLE };
+        testHelper.executeSql(getCreateTypeTableSQL(targetTypes));
+
         new TarantoolTestTypeHelper<>(connection, "test_types", Double.class)
-            .setColumns(TarantoolSqlType.FLOAT, TarantoolSqlType.DOUBLE)
+            .setColumns(targetTypes)
             .setValues(DOUBLE_VALS)
             .testTypes();
     }
 
     @Test
-    void testSetBigDecimal() throws SQLException {
+    void testSetBigDecimalV21() throws SQLException {
```

Comment:
Tarantool will support 'decimal' type natively soon, maybe the test case name will be a bit confusing. The same for testSetBigDecimalV221() below.